src/data/finset/locally_finite.lean

```diff
@@ -515,4 +539,15 @@ lemma image_add_right_Ioo (a b c : α) : (Ioo a b).image (+ c) = Ioo (a + c) (b
 by { simp_rw add_comm _ c, exact image_add_left_Ioo a b c }
 
 end ordered_cancel_add_comm_monoid
+
+@[to_additive] lemma prod_Ioi_mul_eq_prod_off_diag [fintype ι] [linear_order ι]
```

Comment:
What do you think of
```suggestion
@[to_additive] lemma prod_prod_Ioi_mul_eq_prod_prod_off_diag [fintype ι] [linear_order ι]
```

Reply:
Once again, I did not write this lemma. I moved it from one file to another and replaced `filter_lt` by `Ioi`. Nevertheless, this is indeed a more accurate name, so I am happy to replace if you insist.